### PR TITLE
 #169257374 user can provide feedback on accommodation

### PR DESCRIPTION
**Pull request by nshaphan**
#### What does this PR do?
Allows the user to share feedback

#### Description of Task to be completed?
- Create a form to allow the user to share feedback.
- Display feedbacks to the hotel owner

#### How should this be manually tested?
- Clone this repo
- switch to `ft-provide-feedback-on-accommodation-169257374` branch
- Run npm i
- Run npm run watch
- Log in as a requester
- Click `Home` Menu on Navbar 
- Choose a hotel, share a feedback

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#169257374](https://www.pivotaltracker.com/story/show/169257374)

#### Screenshots (if appropriate)
image

#### Questions:
N/A

### CODE REVIEW

**Comment by b0nbon1** [comment]
Hey @nshaphan Great work but I have an issue, I'm not sure if I'm missing a step. When i try to share feedback I get these errors and the spinner keeps spinning :

image

File: src/__tests__/actions/requestSearchAction.spec.js [hunk 1]
```diff
@@ -141,20 +141,20 @@ describe("\"RequestSearchAction\" - Requester", () => {
...
+  //   const expected = [
+  //     actionFunc(LOADING, true),
+  //     actionFunc(REQUEST_SEARCH_ERROR, errorResponse),
+  //     actionFunc(LOADING, false)];
+  //   return store.dispatch(requestSearch({ searchString: "" })).
+  //     then(() => {
+  //       const actual = store.getActions();
+  //       expect(actual).toEqual(expected);
+  //     });
+  // });
```
Comment (b0nbon1) [inline src/__tests__/actions/requestSearchAction.spec.js]: remove the commented lines from the codebase

**Comment by nshaphan** [comment]
> Hey @nshaphan Great work but I have an issue, I'm not sure if I'm missing a step. When i try to share feedback I get these errors and the spinner keeps spinning :
> 
> image

Thanks for the feedback @b0nbon1 , changes have been implemented!

**Comment by nshaphan** [comment]
> The feature works great and I like the look of the feedback panel. One change I think you should make is to remove the toast when a user successfully gives feedback since he can see it on the panel immediately he posts it. Maybe only show it if there was an error.

Thanks for feedback @RaymondMwaura, These changes implemented

**Comment by Billmike** [comment]
@nshaphan resolve conflicts

**Comment by Billmike** [comment]
@nshaphan fix merge conflicts

**Comment by nshaphan** [comment]
> @nshaphan fix merge conflicts

On them now!

**Comment by nshaphan** [comment]
> @nshaphan fix merge conflicts

Conflicts fixed

**Pull request merged**